OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -901,7 +904,7 @@ public IEnumerable<IOrderTargeter> Orders
 			get
 			{
 				yield return new EnterAlliedActorTargeter<BuildingInfo>("ForceEnter", 6,
-					(target, modifiers) => AircraftCanEnter(target) && modifiers.HasModifier(TargetModifiers.ForceMove),
+					(target, modifiers) => AircraftCanEnter(target) && modifiers.HasModifier(TargetModifiers.ForceMove) && Info.CanForceLand,
```

Comment:
Imho the order here should be changed so that we don't have to evaluate `AircraftCanEnter` even if we don't need the result afterwards (for example when `CanForceLand` is false):
`(target, modifiers) => Info.CanForceLand && modifiers.HasModifier(TargetModifiers.ForceMove) && AircraftCanEnter(target),`

Reply:
Done.